packaging/dbscripts/upgrade/pre_upgrade/0000_config.sql

```diff
@@ -858,7 +858,7 @@ select fn_db_add_config_value('BootstrapCommand',
 select fn_db_add_config_value('BootstrapPackageDirectory', '/usr/share/ovirt-host-deploy/interface-3', 'general');
 select fn_db_add_config_value('BootstrapPackageName', 'ovirt-host-deploy.tar', 'general');
 select fn_db_add_config_value('CertExpirationAlertPeriodInDays', '30', 'general');
-select fn_db_add_config_value('CertExpirationWarnPeriodInDays', '120', 'general');
```

Comment:
perhaps more? 2y?  I wonder what others think

Reply:
Do you really want to file a warning into audit_log each day for 2 years (or less if customers are really starts looking at audit_log)?

Reply:
ideally it does automatic renewal before time runs out. I believe it just needs to be
1) large enough so people have time to run through proper maintenance, that's 6-12 months
2) "optimal" proportionally to the certificate lifetime. Considering 5 years it means 3 years without warnings whatsoever, then you are asked to renew(or it happens on upgrades).

I think both 1y and 2y are fine, I just can't tell which one is "better"

Reply:
The better one is 1 year because this is what we already have in the patch. :-)

Reply:
so let's call it 1y